mcloud.yaml

```diff
@@ -10,8 +10,7 @@ integrations:
     git_branch: $UV_BRANCH
     pip_install: poetry==1.7.1
 command: >-
-  cd ultravox && poetry install --no-dev && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train --config_path ultravox/training/configs/$UV_CONFIG
+  cd ultravox && poetry install --no-dev && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train --config_path ultravox/training/configs/llama3_whisper.yaml
```

Comment:
isn't it better to keep the UV_CONFIG env?

Reply:
I explained this separately. I don't know how we can control this from the outside, so it's not really doing anything for us at this point. Originally Justin added this assuming our local envs could control this but that wasn't the case IIRC.